configure.py

```diff
@@ -372,6 +374,8 @@ def binary(name):
 if platform.is_mingw():
     cflags.remove('-fvisibility=hidden');
     ldflags.append('-static')
+    ldflags.append('-g')
+    ldflags.append('-municode')
```

Comment:
What about MinGW+CMake?

Reply:
Isn't that handled from the CMakeLists.txt? I've added the unicode define to the CMakeLists file or am I missing something?

Reply:
Ah right, `-municode` just defines `-DUNICODE` it seems.

Why did you add `-g` though?

Reply:
Seems like the -g flag is unneeded. I'll remove it!

Reply:
`-municode` is unneeded, too, since you're adding `-DUNICODE` above, right?

Reply:
The -municode is required when building with mingw, see https://sourceforge.net/p/mingw-w64/wiki2/Unicode%20apps/, to get the wmain. Isn't the configure.py and cmake script seperate?
I've updated the cmakelists to add the -municode flag when using mingw and the UNICODE flag when building with msvc.

Reply:
Ah I see! We probably should add some (high-level) tests to see if unicode is working as intended and what is *really* needed.

Reply:
I'll look into the tests and see where from which points it's best to test the new additions.